cardscan-base/src/main/res/layout/bouncer_private_activity_scan_card.xml

```diff
@@ -99,56 +117,56 @@
         android:layout_height="75dp"
         android:layout_marginStart="16dp"
         android:layout_marginBottom="16dp"
+        android:contentDescription="@string/an_image_used_only_for_debugging_and_testing"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        android:contentDescription="@string/an_image_used_only_for_debugging_and_testing" />
+        app:layout_constraintStart_toStartOf="parent" />
 
     <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/guideline"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/guideline"
         android:orientation="horizontal"
-        app:layout_constraintGuide_percent="0.5"/>
+        app:layout_constraintGuide_percent="0.5" />
 
     <TextView
         android:id="@+id/cardNumber"
-        android:visibility="invisible"
-        android:textColor="@android:color/white"
-        android:gravity="center"
         android:layout_width="0dp"
         android:layout_height="48dp"
         android:layout_marginStart="16dp"
         android:layout_marginLeft="16dp"
         android:layout_marginTop="0dp"
         android:layout_marginEnd="16dp"
         android:layout_marginRight="16dp"
+        android:gravity="center"
+        android:maxLines="1"
         android:text="@string/_4242_4242_4242_4242"
-        app:autoSizeTextType="uniform"
-        app:autoSizeMinTextSize="12sp"
+        android:textColor="@android:color/white"
+        android:textSize="30sp"
+        android:visibility="invisible"
         app:autoSizeMaxTextSize="100sp"
+        app:autoSizeMinTextSize="12sp"
         app:autoSizeStepGranularity="2sp"
-        android:textSize="30sp"
-        android:maxLines="1"
+        app:autoSizeTextType="uniform"
         app:layout_constraintEnd_toEndOf="@+id/cardRectangle"
         app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="@+id/cardRectangle"
         app:layout_constraintTop_toTopOf="@+id/guideline" />
 
     <TextView
         android:id="@+id/expiry"
-        android:visibility="invisible"
-        android:textColor="@android:color/white"
-        android:gravity="center"
         android:layout_width="0dp"
         android:layout_height="36dp"
         android:layout_marginStart="16dp"
         android:layout_marginLeft="16dp"
         android:layout_marginTop="0dp"
         android:layout_marginEnd="16dp"
         android:layout_marginRight="16dp"
+        android:gravity="center"
+        android:maxLines="1"
         android:text="@string/_5_18"
+        android:textColor="@android:color/white"
         android:textSize="17sp"
-        android:maxLines="1"
+        android:visibility="invisible"
```

Comment:
I know you're just rearranging these properties, but this should probably be `gone` instead of `invisible`